org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.raml

```diff
@@ -0,0 +1,36 @@
+#%RAML 0.8
+title: org.opent2t.sample.hub.superpopular
+version: v1.0
+
+schemas:
+  - Hub: !include org.opent2t.sample.hub.superpopular.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.a"]
+
+/hubResURI:
```

Comment:
This should start with a capital letter for consistency (I think) with the OCF stuff. See [here](https://github.com/openT2T/translators/blob/dev/oic.r.mode/mode.raml) and [here](https://github.com/openT2T/translators/blob/dev/oic.r.switch.binary/binarySwitch.raml) as two examples.